examples/multimodal_v1/utils/nats_queue.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Probably don't need this file anymore. I think this was for the prefill worker queue in V0.